### PR TITLE
update theme-related text in documentation guide

### PR DESCRIPTION
**Pull request by drammock**
closes https://github.com/scientific-python/summit-2024/issues/16

### CODE REVIEW

File: docs/pages/guides/docs.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ parent: Topical Guides
 # Writing documentation
 
 Documentation used to require learning reStructuredText (sometimes referred to
-as reST / RST), but today we have great choices for documentation in markdown,
+as reST / rST), but today we have great choices for documentation in markdown,
```
Comment (drammock) [inline docs/pages/guides/docs.md:14]: (unrelated nitpick)

File: docs/pages/guides/docs.md [hunk 2]
```diff
@@ -29,7 +29,7 @@ and uses the modern MyST plugin to get Markdown support.
 >   based on markdown and HTML. Less support for man pages & PDFs than Sphinx,
 >   since it doesn't use docutils. Has over
 >   [200 plugins](https://github.com/mkdocs/catalog) - they are much easier to
->   write than Sphinx. Examples include [hatch](https://hatch.pypa.io),
+>   write than Sphinx. Example sites include [hatch](https://hatch.pypa.io),
```
Comment (drammock) [inline docs/pages/guides/docs.md:32]: clarification: these are example sites not example plugins; context makes that not super clear.

File: docs/pages/guides/docs.md [hunk 3]
```diff
@@ -147,9 +147,10 @@ several good extensions:
 We are including both possible file extensions. We are also avoiding some common
 file patterns, just in case.
 
-For theme, you have several good options. The clean, light-weight Furo theme is
-shown above. Many scientific packages choose the `sphinx-py-data` theme, which
-is also a good choice (no dark mode, though).
+For theme, many scientific packages choose the
+[pydata-sphinx-theme](https://pydata-sphinx-theme.readthedocs.io/). The
+[Furo theme](https://pradyunsg.me/furo/) is another popular choice. The site
+[sphinx-themes.org](https://sphinx-themes.org/) can be used to compare options.
```
Comment (drammock) [inline docs/pages/guides/docs.md:153]: I also tried to change the `html_theme` to `pydata_sphinx_theme` in the example `conf.py` above, but it appears that pre-commit CI has undone that change. IMO, PST is mature enough now to be shown as the default example. Any guidance on how to get that change to stick?

**Review by drammock — COMMENTED** [review]
self-review

File: docs/pages/guides/docs.md (hunk 3 above)
Comment (henryiii) [reply docs/pages/guides/docs.md:153]: It's from the cookiecutter. The docs are in sync with the template.

I'd rather like a choice if possible, furo is a fantastic, lightweight theme that is very popular for non-scientific libraries. Fine to have the default swap, though.

File: docs/pages/guides/docs.md (hunk 1 above)
Comment (henryiii) [reply docs/pages/guides/docs.md:14]: We can enforce that in pre-commit. ;)

File: docs/pages/guides/docs.md (hunk 3 above)
Comment (henryiii) [reply docs/pages/guides/docs.md:153]: https://github.com/scientific-python/cookie/blob/main/%7B%7Bcookiecutter.project_name%7D%7D/docs/conf.py

File: docs/pages/guides/docs.md (hunk 1 above)
Comment (henryiii) [reply docs/pages/guides/docs.md:14]: Enforced now.

**Review by henryiii — APPROVED** [review]

**Pull request merged**